src/AutoBuild.jl

```diff
@@ -1166,53 +1150,98 @@ function build_jll_package(src_name::String,
 
                 global PATH_list, LIBPATH_list
             """)
+            if !isempty(dependencies)
```

Comment:
This is done already 20 lines above, before init, #844 :slightly_smiling_face:

Reply:
You're right!  I need to get rid of the previous one, because I don't think it will work. :P

We can't do that initialization at compile-time because if we do, we bake in the values of `dep_jll.PATH_list` that existed at compile-time; then if we move artifacts (like as part of a PackageCompiler bundle) we end up with `PATH_list`'s that are out of sync. :(

Reply:
I think I previously convinced you that this could be done at compile time, and now I realize it can't.

Reply:
Yes, you did :joy: ok, we should revert #844, we can't wait for this PR to be merged

Reply:
Done in #882